Sources/HaCWebsiteLib/Controllers/LandingPageController.swift

```diff
@@ -50,4 +36,30 @@ struct LandingPageController {
       Log.error("Socket error occured")
     }
   }
+}
+
+extension Event {
+  func toPostCard() -> PostCard? {
```

Comment:
Awesome that we're thinking ahead in how these events become post cards

Reply:
Might it be useful to have a protocol something like `PostCardRepresentable` which exposes a `postCardRepresentation` var?

Reply:
I like the way you think